spec.html

```diff
@@ -1642,13 +1642,13 @@ <h1>Number::add ( _x_, _y_ )</h1>
               The sum of two infinities of the same sign is the infinity of that sign.
             </li>
             <li>
-              The sum of an infinity and a finite value is equal to the infinite operand.
+              The sum of an infinity and a finite value is equivalent to the infinite operand.
```

Comment:
This is a bit awkward. Why not just this?

```suggestion
              The sum of an infinity and a finite value is the infinite operand.
```